examples/visualization/plot_montage.py

```diff
@@ -0,0 +1,56 @@
+# -*- coding: utf-8 -*-
+"""
+Plotting sensor layouts of EEG Systems
+======================================
+
+Show sensor layouts of different EEG systems.
+
+"""  # noqa: E501
+# Author: Joan Massich <mailsik@gmail.com>
+#
+# License: BSD Style.
+
+from mayavi import mlab
+import os.path as op
+
+import mne
+from mne.channels.montage import get_builtin_montages
+from mne.viz import plot_alignment
+
+# print(__doc__)
+
+data_path = mne.datasets.sample.data_path()
+subjects_dir = op.join(data_path, 'subjects')
+trans_fname = op.join(op.dirname(mne.__file__), "data", "fsaverage",
+                      "fsaverage-trans.fif")
+
+###############################################################################
+# check all montages
+#
+
+fix_units = {'EGI_256':'cm', 'GSN-HydroCel-128':'cm', 'GSN-HydroCel-129':'cm',
```

Comment:
EGI_256 is not in cm, take a look at [this comment](https://github.com/mne-tools/mne-python/pull/5472#issuecomment-483059750): the distance for each channel from the center is exactly 1. for EGI_256.

Reply:
I missed your comment. The way you figured out the distance is smarter than the way I did it. 

The code was pulled from an early stage of #5836 and yes `EGI_256` it's broken.  Now that we are at it, when load `EGI_256` with `eeg=['original', 'projected']` you can see that the original do have a 'brainish' shape but the projected does not.

Either way, we decided with @agramfort to add the example ploting all helmets i a different PR. (I'll ping you there)